src/Components/Home/Home.js

```diff
@@ -9,24 +10,35 @@ import './Home.css';
 
 const Home = ({ favorites }) => {
   const [owls, setOwls] = useState([])
+  const [error, setError] = useState('')
 
   const fetchOwls = useCallback(async () => {
-    const data = await fetchRecentObservations();
-    const owlData = filterForOwls(data);
-    const cleanedOwlData = cleanOwlData(owlData);
-    setOwls(cleanedOwlData);
+    try {
+      const data = await fetchRecentObservations();
+      const owlData = filterForOwls(data);
+      const cleanedOwlData = cleanOwlData(owlData);
+      setOwls(cleanedOwlData);
+    } catch (error) {
+      setError(error.message);
+    }
   }, [])
+  
 
   useEffect(() => {
     fetchOwls();
   }, [fetchOwls])
 
   return (
+    <>
+    {error ? <Redirect push to ='/error' />
+    :(
```

Comment:
I think this is a great example of when you follow solid structure, you can make such a huge change with a single ternary! This Home.js file is super clean, as is the rest of the code in this PR.